src/stream/stream_backend.rs

```diff
@@ -228,11 +228,38 @@ fn create_rtsp_stream(
         .configuration
         .encode
         .clone();
-    let endpoints = &video_and_stream_information.stream_information.endpoints;
+    dbg!(&encode);
+    let endpoint = &video_and_stream_information.stream_information.endpoints[0];
+    dbg!(&endpoint);
+    if endpoint.scheme() != "rtsp" {
+        return Err(SimpleError::new(format!(
+            "The URL's scheme for RTSP endpoints should be \"rtsp\", but was: {:?}",
+            endpoint.scheme()
+        )));
+    } else if endpoint.host_str() != "localhost".into() {
```

Comment:
Else is not necessary after return

Reply:
do you mean like:
```rust
if somethingA { return Err(); }
if somethingB { return Err(); }
if ...
```
instead of:
```rust
if somethingA { return Err(); }
else if somethingB { return Err(); }
else if ...
```
?

Logically the same, indeed cleaner without the elses.

Reply:
yes.

Reply:
Done (053980b)